website/views.py

```diff
@@ -4257,3 +4277,15 @@ def get_context_data(self, **kwargs):
         context["end_date"] = datetime.now().date().isoformat()
 
         return context
+
+
+def create_monitor(request):
+    if request.method == "POST":
+        form = MonitorForm(request.POST)
+        if form.is_valid():
+            form.save()  # This will create a new Monitor instance
+            messages.success(request, "Form submitted successfully!")
+    else:
+        form = MonitorForm()
+
+    return render(request, "Monitor.html", {"form": form})
```

Comment:
Based on the  current `templates` directory naming convention:
```suggestion
    return render(request, "monitor.html", {"form": form})
```